### ISSUE TITLE
Include/Exclude are destructive

### ISSUE THREAD

**Issue by chadhietala** [stefanpenner/broccoli-stew#75]
Given:

```
var globs = ['dummy/**/*.js', 'foo/bar/*.js'];
var app = find(tree, { include: globs, exclude: ['dummy/tests']})

env('!production', function() {
  app = find(tree, {include: globs})
}) 
```

The second `find` will yak because globs is now an array of Minimatch objects.

**Comment by timmfin** [comment]
I assume this would be fixed by using newer version of the broccoli-funnel dependency (via [this PR](https://github.com/broccolijs/broccoli-funnel/pull/14) that may not have gotten out till broccoli-funnel `v0.2.3`).

**Comment by stefanpenner** [comment]
maybe addressed by https://github.com/stefanpenner/broccoli-stew/pull/76

**Comment by stefanpenner** [comment]
#76 does appear to address this

**Comment by stefanpenner** [comment]
should be addressed in the newly released `v0.3.1`

thanks @timmfin for pointing this ou.
